packages/react-ds/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FunctionComponent } from 'react'
+
+import classNames from 'classnames/bind'
+import uniqid from 'uniqid'
+
+import { ISelect } from '@venice/core/models/select.model'
+import styles from '@venice/styles/components/Select.module.scss'
+
+interface ISelectProps extends React.HTMLProps<HTMLSelectElement> {
+  /** React Element */
+  icon?: React.ReactNode | string
+  /** Style */
+  style?: object
+}
+
+const Select: FunctionComponent<ISelectProps & ISelect> = ({
+  id,
+  label,
+  style,
+  className,
+  error,
+  placeholder,
+  children,
+  icon,
+  ...props
+}: ISelectProps & ISelect) => {
+  const selfId = id || uniqid(`select__`)
+  const styleContainer = classNames(
+    styles.container,
+    { [styles.invalid]: Boolean(error) },
+    className
+  )
+
+  return (
+    <div className={styleContainer} style={style}>
+      {label && (
+        <label htmlFor={selfId} className={styles.label}>
+          {label}
+        </label>
+      )}
+      <div className={styles.selectWrapper}>
+        {icon && <span className={styles.icon}>{icon}</span>}
+        <select className={styles.select} id={selfId} {...props}>
+          <option hidden>{placeholder}</option>
+          {children}
+        </select>
+        <div className={styles.caret}>
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 13">
+            <path d="M 23.890625 1.671875 L 23.753906 1.816406 L 13.507812 12.417969 C 13.160156 12.777344 12.640625 13 12.0625 13 C 11.480469 13 10.960938 12.769531 10.613281 12.417969 L 0.375 1.835938 L 0.203125 1.660156 C 0.0742188 1.488281 0 1.285156 0 1.070312 C 0 0.480469 0.554688 0 1.25 0 L 22.855469 0 C 23.550781 0 24.105469 0.480469 24.105469 1.070312 C 24.105469 1.296875 24.023438 1.503906 23.890625 1.671875 Z M 23.890625 1.671875 " />
+          </svg>
```

Comment:
You can add this SVG as a global component (core?) and just import it here (I'll add this caret to SplitButton created by @elizeumatheus)

Reply:
I was thinking to get for the icon lib, but i can do a internal workarround

Reply:
Yes, is better to use a icon lib but as we don't have it yet (😢) this icon is a good example to use in both places.